tests/app/main/test_service_update_audits.py

```diff
@@ -0,0 +1,344 @@
+import mock
+
+from datetime import datetime
+
+from ..helpers import LoggedInApplicationTest
+
+
+class TestServiceUpdates(LoggedInApplicationTest):
+    @mock.patch('app.main.service_update_audits.data_api_client')
+    def test_should_render_activity_page_with_date(self, data_api_client):
+        today = datetime.now().strftime("%d/%m/%Y")
+
+        response = self.client.get('/admin/service-updates')
+        self.assertEquals(200, response.status_code)
+
+        date_header = """
+        <p class="context">
+            Activity for
+        </p>
+        <h1>
+        {}
+        </h1>
+        """.format(today)
+
+        self.assertIn(
+            self._replace_whitespace(date_header),
+            self._replace_whitespace(response.get_data(as_text=True))
+        )
+
+        data_api_client.find_audit_events.assert_called()
+
+    @mock.patch('app.main.service_update_audits.data_api_client')
+    def test_should_render_correct_form_defaults(self, data_api_client):
+        response = self.client.get('/admin/service-updates')
+        self.assertEquals(200, response.status_code)
+
+        self.assertIn(
+            '<input class="filter-field-text" id="audit_date" name="audit_date" type="text" value="">',  # noqa
```

Comment:
You could use `lxml`instead of string comparisons for cases like this. I think some of the other tests might already doing this, since it's listed in the `requirements_for_test`